browser/components/StatsContentCard/StatsContentCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './StatsContentCard.scss';
+import PropTypes from 'prop-types';
+
+
+const StatsContentCard = (props) => {
+  const { godName, imageUrl, stat, label } = props;
+
+  return (
+    <div className="stat-content-card">
+      <img src={imageUrl} alt={`${godName}_image`} />
+      <div className="stat-content-card-info">
+        <h3>{godName}</h3>
+        <p>{label}</p>
```

Comment:
i would really prefer us styling classes instead of h3 or p tags
Maybe 
```
stat-content-card-info-name
stat-content-card-info-stat-type
stat-content-card-info-stat-value
```